spec/models/user_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe User do
+  subject(:user) do
```

Comment:
Just as a remark, you can use curly braces block here instead of `do; end` block